### PR TITLE
Use packet-assisted cluster profile for agent appliance jobs

### PR DESCRIPTION
**Pull request by zaneb**
The appliance is owned by the Ecosystem team, not the installer team.

### CODE REVIEW

File: ci-operator/config/openshift/appliance/openshift-appliance-master.yaml [hunk 1]
```diff
@@ -63,16 +63,7 @@ tests:
...
-      APPLIANCE_IMAGE: agent-preinstall-image-builder
-    env:
-      AGENT_E2E_TEST_BOOT_MODE: DISKIMAGE
-      DEVSCRIPTS_CONFIG: |
-        AGENT_E2E_TEST_BOOT_MODE=DISKIMAGE
-        AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
-        NETWORK_TYPE=OVNKubernetes
-        INSTALLER_PROXY=true
-    workflow: agent-e2e-generic
+    workflow: agent-e2e-appliance-ipv4
```
Comment (andfasano) [inline ci-operator/config/openshift/appliance/openshift-appliance-master.yaml:66]: Shouldn't change the `cluster_profile` also here, for the all config file in this folder?

Comment (zaneb) [reply ci-operator/config/openshift/appliance/openshift-appliance-master.yaml:66]: :man_facepalming: yes

**Pull request merged**